rasa/shared/importers/importer.py

```diff
@@ -464,7 +464,7 @@ async def _get_domain_with_e2e_actions(self) -> Domain:
             [],
             {},
             action_names=[],
-            forms=[],
+            forms={},
```

Comment:
What's the reason for this change?

Reply:
That's related to [this change](https://github.com/RasaHQ/rasa/pull/7171). Seems something went wrong when merging `master` into this branch.